### ISSUE TITLE
Can't install Gulp-Starter

### ISSUE THREAD

**Issue by interglobalmedia** [vigetlabs/gulp-starter#309]
Hi,

I have node and npm installed, I have gulp installed (initially didn't in my project, but when this first failed, I added it), I was not able to install gulp-starter. There is something wrong here. I have npm 3.10.3  and nodejs 6.3.0. Is gulp-starter compatible with those versions? This is the error that I am getting in terminal: 

npm start                                                             ✹

> gulp-starter@3.1.0 start /Users/mariacam/Development/fcc-calculator/workflows
> gulp

module.js:442
    throw err;
    ^

Error: Cannot find module 'require-dir'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/mariacam/Development/fcc-calculator/workflows/gulpfile.js/index.js:14:18)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.3.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! gulp-starter@3.1.0 start: `gulp`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gulp-starter@3.1.0 start script 'gulp'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the gulp-starter package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs gulp-starter
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls gulp-starter
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mariacam/Development/fcc-calculator/workflows/npm-debug.log

**Note: I checked and you have only tested up to nodejs 5.9. There is definitely an issue then with 6.3.0 and maybe even npm 3.10.3!**

**Comment by joemidi** [comment]
Try Node <6.0 not sure if gulp-starter is fully compatible with 6 yet.

**Comment by interglobalmedia** [comment]
I figured that. I guess Gulp-Starter will just have to wait! I think alot of the packages are not compatible with 6.3.0. Looking forward to being able to use it. Thanks!

**Comment by greypants** [comment]
See #302 - you can update all dependencies and it *will* work (confirmed this last week). I haven't yet, because it means dropping support for 0.12.x. But I probably will soon. There's nothing stopping you from doing it though!

**Comment by interglobalmedia** [comment]
I have another issue that popped up after I updated to node 6.3. I lost my global permissions and can't install anything globally. Am trying to figure out how that happened. So, can't do anything until that is resolved because I can't install anything right now -g. These issues all crept up in the last couple of days when I first installed 6.3 then removed and re-installed but now have 4.4.7 and npm 2.15.8. I'm on El Capitan latest version, and I don't know if that has something to do with it. I'm really in the dark with this one.
